cmd/osbuild-composer/main.go

```diff
@@ -107,13 +107,22 @@ func main() {
 
 		composer.InitLocalWorker(l[0])
 	}
-
+	
 	if l, exists := listeners["osbuild-composer-api.socket"]; exists {
 		if len(l) != 1 {
 			logrus.Fatal("The osbuild-composer-api.socket unit is misconfigured. It should contain only one socket.")
 		}
 
-		err = composer.InitAPI(ServerCertFile, ServerKeyFile, config.Koji.EnableTLS, config.Koji.EnableMTLS, config.Koji.EnableJWT, l[0])
+		serverCertFile := config.Koji.ServerCertFile
```

Comment:
The default values are loaded into the config structure in this function: https://github.com/osbuild/osbuild-composer/blob/main/cmd/osbuild-composer/config.go#L86. The returned structure gets overridden by the values in the configuration file. So, instead of checking and overriding here, you should add the default values to the default configuration